clustoapi/server.py

```diff
@@ -82,6 +82,7 @@
     'delete': "curl -X DELETE -s -w '\\nHTTP: %{http_code}\\nContent-type: %{content_type}'",
     'delete_i': "curl -X DELETE -si",
     'head': "curl -s -I",
+    'attr_dict': '[{"key":"group","subkey":"admin","value":"apache"},{"key":"group","subkey":"member","value":"webapp"}]'
```

Comment:
My only complaint is the name of this variable. Should be something clearly indicated is a sample, sample_json_attrs ?